### DISCUSSION TITLE
Add collection icons to the selection list

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5850] [Feature Requests]
In Specify 6, you could assign all collections an icon to make them easily distinguishable from each other. It would be nice to see this in Specify 7.

Here's an example:
<img width="745" alt="image" src="https://github.com/specify/specify7/assets/37256050/60360706-3917-4139-a864-8a8c4693fab0">

**Comment by maxpatiiuk** [comment]
for devs:

`<CustomSelectElement>` could be used to represent a list with icons (like the combo boxes in the mapping view in the workbench mapper)

of course, it won't look anything as clear and tidy as it does right now

you could simply display the table icon right after the radio circle, but having two icons right after each other seems a bit noisy

@grantfitzsimmons would it be acceptable to group collections by discipline, and each group name would have discipline icon next to it?

**Comment by maxpatiiuk** [comment]
besides the difficulty of deciding how to present it, we would have to design the SVG icons for each discipline - no way we are re-introducing the century-old sp6 low res PNGs

**Comment by grantfitzsimmons** [comment]
We already have SVG icons for (most if not all) disciplines represented in the taxon tiles artwork on the home page. I have the originals available upon request and on the SCC Vault.

**Comment by grantfitzsimmons** [comment]
As an FYI– these can be chosen per-collection in the preferences:

<img width="777" alt="image" src="https://github.com/specify/specify7/assets/37256050/8bba62bc-7e38-4a2c-b57b-9f4c1164daa0">
